convene-web/app/views/features/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<nav class="breadcrumbs">
+  <span class="crumb">
+    <%= t('features.title') %>
+  </span>
+</nav>
```

Comment:
@user512 / @anaulin I _think_ a client of mine uses [this gem](https://github.com/fnando/breadcrumbs) for their breadcrumb system; which seems to work pretty well. Thoughts?

Reply:
Never seen that gem before, but it seems like it does what we need and is not overly complicated, so it seems reasonable to use.